README.md

```diff
@@ -13,26 +13,47 @@
 
 [IaSQL](https://iasql.com) lets you model your infrastructure as data by maintaining a 2-way connection between your AWS account and a Postgres SQL database.
 
-## Community and Support
+## ⚡️ Install
 
-- [Community Forum](https://github.com/iasql/iasql-engine/discussions): Best for help with building, discussion about database and infrastructure best practices.
-- [GitHub Issues](https://github.com/iasql/iasql-engine/issues/new): Best for bugs and errors you encounter using IaSQL.
-- [Discord](https://discord.com/invite/machGGczea): Best for sharing what you built and asking about problems with your database or infrastructure.
+To install, visit [Contributing.md](https://github.com/iasql/iasql-engine/blob/main/CONTRIBUTING.md)
```

Comment:
Hmm... I kinda expected this to link the docs? I guess *what* you're installing is actually unclear to me with this being the main repo. Is it the engine or the CLI you'd use to communicate to our SaaS?

Reply:
ops that is a typo